src/cpp/desktop/DesktopBrowserWindow.cpp

```diff
@@ -68,10 +68,6 @@ BrowserWindow::BrowserWindow(bool showToolbar,
    setUnifiedTitleAndToolBarOnMac(true);
 
    desktop::enableFullscreenMode(this, false);
-
-   QShortcut* copyShortcut = new QShortcut(QKeySequence::Copy, this);
```

Comment:
@jjallaire I've tested this fairly thoroughly and it seems like copy still works as expected, but it would be good to have some more testing post-merge. Note that removing this implies that instead of directly emitting a copy event on Ctrl + C, we instead do the dance of having Qt emit a Ctrl + C key combination, and that being handled on the client side as a copy (or other action, in the case of Emacs mode).

Would we be able to borrow some time from @dfalty to test this some more this post-merge?